### PR TITLE
Targeting the correct main now

### PR DESCRIPTION
**Pull request by VictorsCodingCorner**
setting up Jasmine testing. First step in ensuring the pipeline will run tests when making a pull request.

Created new pull request to target the correct main branch

### CODE REVIEW

**Review by sebgro98 — DISMISSED** [review]
Approved

**Review by Badde00 — APPROVED** [review]
Approved

File: .github/workflows/test.yml
Comment (amos1969) [inline .github/workflows/test.yml:15]: Recommendation is to use v4 of actions/checkout now: [ uses: actions/checkout@v4](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#example-using-versioned-actions)

**Review by sebgro98 — APPROVED** [review]
This has been updated now and waiting for approval

**Review by Badde00 — APPROVED** [review]
Approved with updates

**Comment by amos1969** [comment]
Approved

**Pull request merged**
